nautobot/utilities/testing/api.py

```diff
@@ -375,6 +375,7 @@ class CreateObjectViewTestCase(APITestCase):
         create_data = []
         validation_excluded_fields = []
         slug_source = None
+        slugify_function = staticmethod(slugify)
```

Comment:
I added the staticmethod because otherwise it becomes a bound method, and then `self.slugify_function("string")` becomes a call to `slugify(self, "string")` which is of course wrong. Open to suggestions as to an alternate approach here but this was the simplest fix I found.

Reply:
Omitting `staticmethod` seems to work for `print()`, but not for most other functions that I've tried (including `slugify` and our `slugify_dashes_to_underscores` and `slugify_dots_to_dashes` functions) as they get confused by the extra `self` parameter. OK if I leave this as-is?